proto/osmosis/lockup/tx.proto

```diff
@@ -42,5 +42,10 @@ message MsgBeginUnlockingAllResponse { repeated PeriodLock unlocks = 1; }
 message MsgBeginUnlocking {
   string owner = 1 [ (gogoproto.moretags) = "yaml:\"owner\"" ];
   uint64 ID = 2;
+  // Amount of unlocking coins. Unlock all if not set.
+  repeated cosmos.base.v1beta1.Coin coins = 3 [
```

Comment:
AFAIK 0 element in repeated field is encoded same with empty message in optional field, and optional is default on proto3. The new proto code will find out there is no `coins = 3` field in the existing messages, so it will set it to nil(or empty sdk.Coins{})

Reply:
oh cool, your right! We just need to test on a front-end on the testnet, that the current flow doesn't break